lua/nvim-surround/html.lua

```diff
@@ -13,8 +13,9 @@ end
 ---@param include_brackets? boolean Whether or not to include the angle brackets.
 ---@return string[][]? @The HTML tag pair.
 M.get_tag = function(include_brackets)
-    local input = vim.fn.input({ prompt = "Enter an HTML tag: " })
-    if not input then
+    -- Handle cancellation by the user
+    local ok, input = pcall(vim.fn.input, { prompt = "Enter an HTML tag: " })
+    if not ok or not input then
```

Comment:
Yes, I can enter an empty string then but because of this code we still return early: 
```lua
local element = input:match("^<?([%w-]+)")
-- [...]
if not element then
  return nil
end
```

Reply:
Hmmm.... I'll merge this in for now, and then see if removing that piece of code fixes #54